config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
+worker_shutdown_timeout 240s;
```

Comment:
I'm currently basing this timeout value on the default [connection draining](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/config-conn-drain.html) / [deregistration delay](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#deregistration-delay) of 300 seconds.

Reply:
Noted. We also already do this in `main.conf` ...

https://github.com/Intellection/docker-nginx-proxy/blob/b87f59bfe036dff06870fd7afb854ebc22aac108/config/main.conf#L3

Reply:
I've moved the configuration to `main.conf`.